app/views/static/donors.html.erb

```diff
@@ -4,8 +4,8 @@
       <div class="Opciones donors">
           <div>IMPACTO<span class="info_boton">Conocé los beneficios de convertirte en donante VOLUNTARIO Y HABITUAL</span></div>
           <div>REQUISITOS<span class="info_boton">Ténes dudas de si podes donar? Lee las condiciones que tiene que tener un DONANTE</span></div>
-          <div>CONSEJOS<span class="info_boton">Hace que tu donación se convierta en un acto SEGURO, CONFORTABLE Y GRATIFICANTE</span></div>
-          <div>MÁS INFO<span class="info_boton">Querés informarte más del tema? Te ofrecemos links de interés sobre la donación de sangre.</span></div>
+          <div style="margin-left:30%">CONSEJOS<span class="info_boton">Hace que tu donación se convierta en un acto SEGURO, CONFORTABLE Y GRATIFICANTE</span></div>
```

Comment:
Si, igualmente eso es provisorio

El 14 de octubre de 2015, 3:17 p. m., Fernando Fajardo<
notifications@github.com> escribió:

> In app/views/static/donors.html.erb
> <https://github.com/OurMark/hermanosdesangre/pull/41#discussion_r42031370>
> :
>
> > @@ -4,8 +4,8 @@
> >        <div class="Opciones donors">
> >            <div>IMPACTO<span class="info_boton">Conocé los beneficios de convertirte en donante VOLUNTARIO Y HABITUAL</span></div>
> >            <div>REQUISITOS<span class="info_boton">Ténes dudas de si podes donar? Lee las condiciones que tiene que tener un DONANTE</span></div>
> > -          <div>CONSEJOS<span class="info_boton">Hace que tu donación se convierta en un acto SEGURO, CONFORTABLE Y GRATIFICANTE</span></div>
> > -          <div>MÁS INFO<span class="info_boton">Querés informarte más del tema? Te ofrecemos links de interés sobre la donación de sangre.</span></div>
> > +          <div style="margin-left:30%">CONSEJOS<span class="info_boton">Hace que tu donación se convierta en un acto SEGURO, CONFORTABLE Y GRATIFICANTE</span></div>
>
> @CarolinaIungelsonGlobant <https://github.com/CarolinaIungelsonGlobant>
> porfa evitemos estilos inline, lo podes corregir?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OurMark/hermanosdesangre/pull/41/files#r42031370>.
>

-- 


The information contained in this e-mail may be confidential. It has been 
sent for the sole use of the intended recipient(s). If the reader of this 
message is not an intended recipient, you are hereby notified that any 
unauthorized review, use, disclosure, dissemination, distribution or 
copying of this communication, or any of its contents, 
is strictly prohibited. If you have received it by mistake please let us 
know by e-mail immediately and delete it from your system. Many thanks.

 

La información contenida en este mensaje puede ser confidencial. Ha sido 
enviada para el uso exclusivo del destinatario(s) previsto. Si el lector de 
este mensaje no fuera el destinatario previsto, por el presente queda Ud. 
notificado que cualquier lectura, uso, publicación, diseminación, 
distribución o copiado de esta comunicación o su contenido está 
estrictamente prohibido. En caso de que Ud. hubiera recibido este mensaje 
por error le agradeceremos notificarnos por e-mail inmediatamente y 
eliminarlo de su sistema. Muchas gracias.